.eslintrc.json

```diff
@@ -29,15 +29,15 @@
     ],
     "no-tabs": "error",
     "react/prop-types": "off", // Props should be described as TS types
-    "@typescript-eslint/explicit-function-return-type": "off",
-    "@typescript-eslint/no-var-requires": "warn"
+    "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
any particular reason for this to be disabled?

Reply:
if it complicates React code unreasonably and hurts readability - should we just omit returns types everywhere for React? what are the best practices here?

Reply:
Best practices is to make TS strict as possible to use all advantages of it, so you are right! I removed this rule and refactored all warnings. Furthermore I made tsconfig more stricter.